--- duktape-android/duktape/src/main/jni/duktape/duk_config.h	2018-09-23 13:57:03.902152500 +0300
+++ duktape-android/duktape/src/main/jni/duktape/duk_config.h	2018-09-23 14:01:01.088250000 +0300
@@ -1884,6 +1884,13 @@
 #error no provider for DUK_USE_DATE_GET_NOW()
 #endif
 
+/** Duktape-Android modification: Hack in date parsing and timezone support for Android */
+extern duk_int_t android__get_local_tzoffset(duk_double_t time);
+extern duk_bool_t android__date_parse_string(duk_context* ctx, const char* str);
+#define DUK_USE_DATE_GET_LOCAL_TZOFFSET android__get_local_tzoffset
+#define DUK_USE_DATE_PARSE_STRING android__date_parse_string
+/** End Duktape-Android. */
+
 #if defined(DUK_USE_DATE_GET_LOCAL_TZOFFSET)
 /* External provider already defined. */
 #elif defined(DUK_USE_DATE_TZO_GMTIME_R) || defined(DUK_USE_DATE_TZO_GMTIME_S) || defined(DUK_USE_DATE_TZO_GMTIME)
